api/debug/doc.go

```diff
@@ -18,8 +18,21 @@
 // This file is derived from internal/debug/api.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// Package debug interfaces Go runtime debugging facilities.
-// This package is mostly glue code making these facilities available
-// through the CLI and RPC subsystem. If you want to use them from Go code,
-// use package runtime instead.
+/*
+Package debug interfaces Go runtime debugging facilities.
```

Comment:
How about changing like below? Or shouldn't it be something like "interface A with B"?
```suggestion
Package debug provides interfaces for Go runtime debugging facilities.
```